packages/api/src/routes/internal/medical/docs.ts

```diff
@@ -381,10 +382,18 @@ router.post(
     const triggerConsolidated = getFromQueryAsBoolean("triggerConsolidated", req);
     const cxDocumentRequestMetadata = cxRequestMetadataSchema.parse(req.body);
 
+    // TODO ENG-618: Get patients first facility id if no facility id is provided
+    const patientFacilityId = facilityId
+      ? facilityId
+      : await getPatientPrimaryFacilityIdOrFail({
+          cxId,
+          patientId,
+        });
```

Comment:
Can't we just make it required since it's an internal endpoint? Sure, need to check where it's used, but it seems under our control and it avoids that we call the endpoint w/o providing the facility and end up querying HIEs w/ the wrong NPI.